src/utilities/graphql/operations.ts

```diff
@@ -12,8 +12,21 @@ function isOperation(
  * Determine if a document is a mutation document.
  *
  * @param document - The GraphQL document to check
+ * @returns A boolean indicating if the document is a mutation operation
  *
- * @since 3.8.0
+ * @example
+ *
+ * ```ts
+ * import { isMutationOperation } from "@apollo/client/utilities";
+ *
+ * const mutation = gql`
+ *   mutation MyMutation {
+ *     # ...
+ *   }
+ * `;
+ *
+ * isMutationOperation(mutation); // true
+ * ```
```

Comment:
```suggestion
 * ```
 * 
 * @remark
 * If you are authoring a link, you might not need to use this utility.
 * Use the `operationType` passed into the link as a property on the `operation` argument instead.
```

Do you think it makes sense to add this remark to all three functions?